packages/editor/src/panels/files/toolbar.tsx

```diff
@@ -345,6 +343,11 @@ export function PanelToolbar({
 }) {
   const { t } = useTranslation()
   const { createNewFolder } = useCurrentFiles()
+  const filesState = useMutableState(FilesState)
+  const originalPath = useMutableState(EditorState).projectName.value
+  const showBackButton =
+    filesState.selectedDirectory.value.replace(/^\/|\/$/g, '').split('/').length >
```

Comment:
add this regex in the common file https://github.com/ir-engine/ir-engine/blob/95ab2a5dd0b2ae513c810a41c48897cfd9cd7cdd/packages/common/src/regex/index.ts